src/main/kotlin/org/jitsi/jibri/util/testHelpers/Eventually.kt

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
i originally had it in the tests, but i moved it out for some reason.  maybe it was just because i thought it looked a little confusing nestled amongst the tests in its own directory (where normally the test dirs exactly mirror the real test files).  but that doesn't seem like a big deal, so moved it back to the test dir.